docs/src/userguide/source_files_and_compilation.rst

```diff
@@ -854,6 +854,20 @@ Cython code.  Here is the list of currently supported directives:
     signature, which cannot otherwise be retrieved after
     compilation.  Default is False.
 
+``embedsignature.format`` (``c`` / ``python`` / ``clinic``)
+    If set to ``c``, Cython will generate signatures preserving
+    C type declarations and Python type annotations. If set to
+    ``python``, Cython will do a its best attempt to use pure-Python
+    type annotations in embedded signatures. For arguments without
+    Python type annotations, the C type is mapped to the closest
+    Python type equivalent (e.g., C ``short`` is mapped to Python
+    ``int`` type and C ``double`` is mapped to Python ``float`` type).
+    The specific output and type mapping are experimental and may
+    change over time.  If set to ``clinic`` and the ``binding``
+    directive is set to False, Cython will generate signatures
+    compatible with CPython's Argument Clinic. Default is
+    ``c``.
```

Comment:
So, what happens if `emberdignature.format=clinic` and `binding=True`?

Reply:
No signature is generated, because CythonFunction does not support the  `__text_signature__` descriptor like CPython's `builtin_function_or_method`.

How would you like to rephrase the documentation?

As I told you before, this `clinic` format is of little use in Cython 3.

Reply:
> As I told you before, this `clinic` format is of little use in Cython 3.

Perhaps a stupid question, but why include it at all then?

Reply:
Personally I'd say "The clinic format generates signatures that can be understood by CPython's argument clinic tool and used to generate the `__text_signature__` attribute.  This mainly useful when `binding=False`, since the functions generated with `binding=True` do not have a `__text_signature__` attribute."

I'm not sure I'd disable it when `binding=True` (even if it's a bit pointless there).

Reply:
"little use" does not mean "no use". Those that for whatever reason want to set `binding=False` may find the new option useful to get `__text_signature__` and then the `inspect` module to work closer to CPython builtin functions and methods.